evm/src/Manager.sol

```diff
@@ -104,9 +107,15 @@ abstract contract Manager is
         mode = _mode;
         chainId = _chainId;
         evmChainId = block.chainid;
+        // save the deployer (check this on iniitialization)
+        deployer = _msgSender();
```

Comment:
Any reason in particular why we're using the Context package here now but still msg.sender everywhere else?

Reply:
to cover the case in which the deployer is a contract to retrieve the address who initiated the deployment.

Reply:
We don't override it anywhere though right so it's still `msg.sender`?

Reply:
You're correct, I'll make the change.